app/_src/kubernetes-ingress-controller/reference/combined-services-from-different-httproutes.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Combined Services From Different HTTPRoutes 
+type: reference
```

Comment:
I'm not 100% convinced if reference is the best place for this.
Is this maybe a doc in observability that is more generic. "How services gets translated" and we explain both methods? I would love to hear @lena-larionova 's opinion on this :)

Reply:
IMO this page (and the potential "How services are translated") is more suitable to be in references than observability. I think the "observability" directory is for pages for introducing how could users know "What happens in KIC", while the page is more likely to introduce how KIC works.